### DISCUSSION TITLE
probabilities for each class

### DISCUSSION THREAD

**Discussion by ArkhamKnight-ZZZ** [open-mmlab/mmdetection#9117] [Q&A] [answered]
I used faster-rcnn network for target detection, how can I obtain the probabilities distribution for each class？
for example, the model detects the object, and what I want is a dict in the following format:
{'cat':0.05, 'dog':0.78, 'tiger':0.17}
how can I get this result?

**Comment by ZwwWayne** [comment, marked as answer]
The output of the model already contains the information. Its format is List[List[boxes, scores]]], where the outer list is for classes and the inner list indicates the number of boxes of that class. The box has 5 dimension where the 5th dimension is the score.
